viewer/viewer.js

```diff
@@ -2161,6 +2161,7 @@ async function premain () {
   Db.initialize({
     host: internals.elasticBase,
     prefix: internals.prefix,
+    queryExtraIndices: Config.getArray('queryExtraIndices', ''),
```

Comment:
`Db.initialize` now needs to be passed the `queryExtraIndices` array value from config, defaulting to empty if not found.